lib/boilerplate.js

```diff
@@ -301,6 +302,14 @@ var boilerplate = function (gulp, opts) {
 
   gulp.task('default', [opts.watch ? 'watch' : 'once']);
 
+  gulp.task('nsp', function (cb) {
+  gulpNSP({
+    shrinkwrap: __dirname + '/../npm-shrinkwrap.json',
```

Comment:
This needs to use `path` so that it will work on systems that have different separators.

```js
shrinkwrap: path.resolve(__dirname, '..', 'npm-shrinkwrap.json')
```